views/layout.erb

```diff
@@ -6,9 +6,16 @@
   <meta name="viewport" content="width=device-width,initial-scale=1">
   <link href="/css/lib/basscss.min.css" rel="stylesheet">
 </head>
-<body>
-  <div class="container">
+<body class="bg-darken-1">
+  <div class="h5 px2 py1 bg-navy white center">
+    <img src="/img/usa-flag.png">&nbsp;
+    An official website of the United States Government
+  </div>
+  <div class="py2 mb3 bg-white center">
+    <img src="/img/seal-dhs.svg" class="align-middle" width="100">
```

Comment:
ah, didn't think about that -- i replaced with a mock seal for the time being. thanks for comment.